core/src/main/java/org/apache/gravitino/Configs.java

```diff
@@ -322,4 +323,11 @@ private Configs() {}
           .version(ConfigConstants.VERSION_0_5_1)
           .intConf()
           .createWithDefault(DEFAULT_METRICS_TIME_SLIDING_WINDOW_SECONDS);
+
+  public static final ConfigEntry<Optional<String>> REST_API_EXTENSION_PACKAGES =
+      new ConfigBuilder("gravitino.extension.restApiPackages")
```

Comment:
How about "gravitino.server.rest.extensionPackages"? Also, can you make this config `toSequence` so you don't have to split it yourself.